staging/src/kubevirt.io/client-go/api/v1/types.go

```diff
@@ -1220,6 +1220,15 @@ type KubeVirtSpec struct {
 
 	// The ImagePullPolicy to use.
 	ImagePullPolicy k8sv1.PullPolicy `json:"imagePullPolicy,omitempty" valid:"required"`
+
+	// The namespace the Prometheus ServiceMonitor
```

Comment:
IIUC this is the namespace of Prometheus, not the ServiceMonitor (which we install in our own namespace atm)?

Reply:
You are correct! That wording/naming can certainly be improved.